lib/internal/abort_controller.js

```diff
@@ -309,20 +309,21 @@ function abortSignal(signal, reason) {
 }
 
 class AbortController {
-  #signal = createAbortSignal();
+  #signal = undefined;
 
   /**
    * @type {AbortSignal}
    */
   get signal() {
+    this.#signal ??= createAbortSignal();
     return this.#signal;
   }
 
   /**
    * @param {any} reason
    */
   abort(reason = new DOMException('This operation was aborted', 'AbortError')) {
-    abortSignal(this.#signal, reason);
+    abortSignal(this.signal, reason);
```

Comment:
Let's not make it susceptible to prototype pollution
```suggestion
    abortSignal(this.#signal ??= createAbortSignal(), reason);
```